### PR TITLE
Hmis 950 dev

### PR DESCRIPTION
**Pull request by mjotichmcts**
### Change description ###
Updated 'occurred' and added HMI transaction code + SNow  description contains endpoint information

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by subrotomukherjee — APPROVED** [review]

File: infrastructure/template/api-op-create-resource-user-policy.xml [hunk 1]
```diff
@@ -111,10 +111,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Create User: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-create-resource-user-policy.xml:117]: Gatiway should be Gateway

File: infrastructure/template/api-op-create-vh-policy.xml [hunk 2]
```diff
@@ -89,10 +89,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Create VH: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-create-vh-policy.xml:95]: Gatiway should be Gateway

File: infrastructure/template/api-op-delete-hearing-policy.xml [hunk 3]
```diff
@@ -118,10 +118,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Delete Hearing: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-delete-hearing-policy.xml:124]: Gatiway should be Gateway

File: infrastructure/template/api-op-delete-participant-policy.xml [hunk 4]
```diff
@@ -90,10 +90,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Delete Participant: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-delete-participant-policy.xml:96]: Gatiway should be Gateway

File: infrastructure/template/api-op-delete-vh-by-id-policy.xml [hunk 5]
```diff
@@ -89,10 +89,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Delete VH: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-delete-vh-by-id-policy.xml:95]: Gatiway should be Gateway

File: infrastructure/template/api-op-edit-participant-policy.xml [hunk 6]
```diff
@@ -89,10 +89,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update Participant: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-edit-participant-policy.xml:95]: Gatiway should be Gateway

File: infrastructure/template/api-op-get-people-by-params-elinks-policy.xml [hunk 7]
```diff
@@ -126,10 +126,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Get People: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-get-people-by-params-elinks-policy.xml:132]: Gatiway should be Gateway

File: infrastructure/template/api-op-get-sessions-policy.xml [hunk 8]
```diff
@@ -182,10 +182,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Get Sessions: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-get-sessions-policy.xml:188]: Gatiway should be Gateway

File: infrastructure/template/api-op-request-direct-hearing-policy.xml [hunk 9]
```diff
@@ -110,10 +110,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Request Direct Hearing: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-request-direct-hearing-policy.xml:116]: Gatiway should be Gateway

File: infrastructure/template/api-op-request-hearing-policy.xml [hunk 10]
```diff
@@ -118,10 +118,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Request Hearing: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-request-hearing-policy.xml:124]: Gatiway should be Gateway

File: infrastructure/template/api-op-retrieve-vh-by-id-policy.xml [hunk 11]
```diff
@@ -89,10 +89,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Retrieve VH by ID : " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-retrieve-vh-by-id-policy.xml:95]: Gatiway should be Gateway

File: infrastructure/template/api-op-retrieve-vh-by-user-policy.xml [hunk 12]
```diff
@@ -131,10 +131,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Retrive VH by User: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-retrieve-vh-by-user-policy.xml:137]: Gatiway should be Gateway

File: infrastructure/template/api-op-update-hearing-policy.xml [hunk 13]
```diff
@@ -144,10 +144,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update Hearing: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-update-hearing-policy.xml:150]: Gatiway should be Gateway

File: infrastructure/template/api-op-update-listing-policy.xml [hunk 14]
```diff
@@ -111,10 +111,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update Listing: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-update-listing-policy.xml:117]: Gatiway should be Gateway

File: infrastructure/template/api-op-update-resource-location-policy.xml [hunk 15]
```diff
@@ -117,10 +117,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update Location: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-update-resource-location-policy.xml:123]: Gatiway should be Gateway

File: infrastructure/template/api-op-update-resource-user-policy.xml [hunk 16]
```diff
@@ -119,10 +119,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update User: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-update-resource-user-policy.xml:125]: Gatiway should be Gateway

File: infrastructure/template/api-op-update-vh-by-id-policy.xml [hunk 17]
```diff
@@ -89,10 +89,10 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI error when processing transaction: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
+							new JProperty("description", "HMI - Update VH By ID: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
-							new JProperty("short_description", "HMI - API Gateway Error"),
+							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-update-vh-by-id-policy.xml:95]: Gatiway should be Gateway

File: infrastructure/template/api-op-add-participant-policy.xml [hunk 18]
```diff
@@ -89,7 +89,7 @@
 							new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
 							new JProperty("category", "Data Issue"),
 							new JProperty("contact_type", "Alert"),
-							new JProperty("description", "HMI - Testing Error Handling"),
+							new JProperty("description", "HMI - Add Participant: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
 							new JProperty("impact", "2"),
 							new JProperty("service_offering", "138e0c541bc5bc507bdaddf0b24bcb2a"),
 							new JProperty("short_description", "HMI - API Gatiway Error"),
```
Comment (ShufenZ) [inline infrastructure/template/api-op-add-participant-policy.xml:95]: Gatiway should be Gateway

**Pull request merged**